### PR TITLE
DEV-3516-feat(piscine-go): add README.md for new go exercise shoppingsummaryco…

### PR DESCRIPTION
**Pull request by tiagocollot**
…unter

Created and added README.md for new go exercise shoppingsummarycounter

### CODE REVIEW

**Review by jotapero — APPROVED** [review]
Everything in order

File: subjects/shoppingsummarycounter/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package main
+
+import (
+  "fmt"
+  "piscine"
+)
+
+func main() {
+  summary := "Burger Water Carrot Coffee Water Water Chips Carrot Carrot Burger Carrot Water"
+    for index,element := range piscine.ShoppingSummaryCounter(summary){
```
Comment (eslopfer) [inline subjects/shoppingsummarycounter/README.md:31]: After a `,` there should be a space. Try putting this into a go  file and run gofmt. There should be a space before the opening curly brace as well.

File: subjects/shoppingsummarycounter/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      fmt.Println(index,"=>",element)
```
Comment (eslopfer) [inline subjects/shoppingsummarycounter/README.md:32]: Same as above

**Review by eslopfer — CHANGES_REQUESTED** [review]

File: subjects/shoppingsummarycounter/README.md (hunk 1 above)
Comment (jotapero) [reply subjects/shoppingsummarycounter/README.md]: fixed

File: subjects/shoppingsummarycounter/README.md (hunk 2 above)
Comment (jotapero) [reply subjects/shoppingsummarycounter/README.md]: fixed

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**